cmd/kubeadm/app/preflight/checks.go

```diff
@@ -336,7 +336,9 @@ func (sysver SystemVerificationCheck) Check() (warnings, errors []error) {
 	for _, v := range validators {
 		warn, err := v.Validate(system.DefaultSysSpec)
 		errs = append(errs, err)
-		warns = append(warns, warn)
+		if warn != nil {
```

Comment:
Why do we need this here but not for `err`?

It looks like we accumulate into `errs` regardless, filter with `utilerrors.NewAggregate` and then check `err` below.  We should probably have the check here for both `err` and `warn`, not use NewAggregate and check `len(err)` below. 

`utilerrors.NewAggregate` prints stuff out in a way that is different from how we are printing out other lists of errors.  We should just have one flat list.